optest/optest_test.go

```diff
@@ -221,3 +254,31 @@ func testGenerateState(t *testing.T) string {
 
 	return state
 }
+
+func TestNew(t *testing.T) {
+	testUsers := map[string]TestUser{
+		"foo": {
+			Audience:           "foo-client",
+			Subject:            "foo",
+			Name:               "Foo Bar",
+			GivenName:          "Foo",
+			FamilyName:         "Bar",
+			Locale:             "en-US",
+			Email:              "foo@bar.com",
+			AccessTokenKeyType: "JWT+AT",
+			IdTokenKeyType:     "JWT",
+		},
+	}
+
+	_, err := New()
+	require.NoError(t, err)
+
+	_, err = New(WithTestUsers(nil))
+	require.ErrorContains(t, err, "at least one test user is required")
+
+	_, err = New(WithTestUsers(testUsers))
+	require.ErrorContains(t, err, "the DefaultTestUser \"test\" could not be find in TestUsers")
```

Comment:
```suggestion
	require.ErrorContains(t, err, "the DefaultTestUser \"test\" could not be found in TestUsers")
```
(if making my corresponding suggested change above)